runtime/kusama/src/lib.rs

```diff
@@ -1036,15 +1198,54 @@ impl pallet_babe::migrations::BabePalletPrefix for Runtime {
 	}
 }
 
-pub struct BabeEpochConfigMigrations;
-impl frame_support::traits::OnRuntimeUpgrade for BabeEpochConfigMigrations {
+pub struct ParachainHostConfigurationMigration;
+impl frame_support::traits::OnRuntimeUpgrade for ParachainHostConfigurationMigration {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		pallet_babe::migrations::add_epoch_configuration::<Runtime>(
-			babe_primitives::BabeEpochConfiguration {
-				allowed_slots: babe_primitives::AllowedSlots::PrimaryAndSecondaryPlainSlots,
-				..BABE_GENESIS_EPOCH_CONFIG
-			}
-		)
+		let config = parachains_configuration::HostConfiguration {
+			max_code_size: MaxCodeSize::get(),
+			max_head_data_size: MaxHeadSize::get(),
+			max_upward_queue_count: 10,
+			max_upward_queue_size: 50 * 1024,
+			max_upward_message_size: 50 * 1024,
```

Comment:
2.6 GB ....? Or is queue_size in bytes and this means we can only store 1 maximal sized message? The queue_count above could be a hint that it is the latter.

Reply:
even if it is the latter, we likely don't want a single message to occupy all our capacity?

Reply:
> Or is queue_size in bytes and this means we can only store 1 maximal sized message? 

This is it

> we likely don't want a single message to occupy all our capacity?

why not? I figure if a parachain wants to use all their capacity in a single message then they ought to be able to